File: src/main/java/gargoyle/ct/prop/impl/CTSimpleSerializableProperty.java
package gargoyle.ct.prop.impl;

import java.io.Serializable;

public class CTSimpleSerializableProperty<T extends Serializable> extends CTSimpleProperty<T> {
    public CTSimpleSerializableProperty(String name) {
        this(name, null);
    }

    public CTSimpleSerializableProperty(String name, T def) {
        super(name, def);
    }
}

File: util/src/main/kotlin/gargoyle/ct/util/prop/impl/simple/CTSimpleSerializableProperty.kt
package gargoyle.ct.util.prop.impl.simple

import kotlin.reflect.KClass

class CTSimpleSerializableProperty<T : Any> : CTSimpleProperty<T> {
    constructor(type: KClass<T>, name: String) : super(type, name)
    constructor(name: String, def: T) : super(name, def)
}
